site/landing/layouts/index.html

```diff
@@ -293,65 +258,15 @@ <h2 class="sr-only">Join the Project</h2>
-        <path class="right item item--outer"
-          d="M82.5 48c.8 0 1.5.7 1.5 1.5v9c0 .8-.7 1.5-1.5 1.5H69V48h13.5zm0-24c.8 0 1.5.7 1.5 1.5v9c0 .8-.7 1.5-1.5 1.5H69V24h13.5z" />
-        <path class="left item item--outer"
-          d="M15 48v12H1.5C.7 60 0 59.3 0 58.5v-9c0-.8.7-1.5 1.5-1.5H15zm0-24v12H1.5C.7 36 0 35.3 0 34.5v-9c0-.8.7-1.5 1.5-1.5H15z" />
-        <path class="top item item--outer"
-          d="M34.5 0c.8 0 1.5.7 1.5 1.5V15H24V1.5c0-.8.7-1.5 1.5-1.5h9zm24 0c.8 0 1.5.7 1.5 1.5V15H48V1.5c0-.8.7-1.5 1.5-1.5h9z" />
-        <path class="bottom item item--outer"
-          d="M60 69v13.5c0 .8-.7 1.5-1.5 1.5h-9c-.8 0-1.5-.7-1.5-1.5V69h12zm-24 0v13.5c0 .8-.7 1.5-1.5 1.5h-9c-.8 0-1.5-.7-1.5-1.5V69h12z" />
-      </g>
-    </svg>
-
-    <nav class="footer-nav" aria-label="Full">
-      <ul class="footer-nav__list">
-        <li class="footer-nav__item">
-          <a href="/book/doc/introduction.html">Documentation</a>
-        </li>
-        <li class="footer-nav__item">
-          <a href="/guides/getting_started">Getting&nbsp;Started</a>
-        </li>
-        <li class="footer-nav__item">
-          <a href="#partners">Partners</a>
-        </li>
-        <li class="footer-nav__item">
-          <a href="{{ .Site.Params.github_repo }}">Source</a>
-        </li>
-        <li class="footer-nav__item">
-          <a href="{{ .Site.Params.one_year_url }}">Blog</a>
-        </li>
-        <li class="footer-nav__item footer-nav__item--icons">
-          <a href="{{ .Site.Params.github_repo }}" aria-label="View on Github">
-            <svg focusable="false" width="24" height="20" viewBox="0 0 24 20">
-              <path fill="currentColor" fill-rule="evenodd"
-                d="M11.4 1A9 9 0 002 10c0 4.2 2.7 7.7 6.5 9 .5 0 .7-.2.7-.5v-2.2s-2.7.5-3.3-1.2c0 0-.4-1-1-1.3 0 0-1-.6 0-.6 0 0 1 0 1.5 1 .8 1.4 2.2 1 2.8.7 0-.6.3-1 .6-1.2-2.1-.2-4.3-.5-4.3-4.1 0-1 .3-1.6.9-2.2C6.3 7 6 6 6.5 4.9c.8-.3 2.6 1 2.6 1a9.3 9.3 0 014.9 0s1.8-1.3 2.6-1c.6 1.2.2 2.2.1 2.5.6.6 1 1.2 1 2.2 0 3.6-2.2 3.9-4.4 4 .4.4.7 1 .7 1.8v3.1c0 .3.1.6.6.5a9.3 9.3 0 006.5-9c0-5-4.4-9-9.7-9z" />
-            </svg>
-          </a>
-        </li>
-      </ul>
-    </nav>
-
-    <p>© 2019 &ndash; {{ now.Format "2006" }} lowRISC contributors, <a href="/privacy-policy/">Privacy
-      Policy</a>, <a href="/usage-policy/">Usage Policy</a>.</p>
-    <p>The text content on this website is licensed under a
-    <a href="https://creativecommons.org/licenses/by-sa/4.0/">Creative Commons
-      Attribution 4.0 International License</a>, except where otherwise noted.</p>
-    <p>No license is granted for the OpenTitan logo or other trademarks. <a
-      href="/trademark-usage-policy/">Trademark Usage Policy</a>.</p>
-  </footer>
+  {{ partial "footer.html" . }}
 
   <script src="/js/tiny-slider.js"></script>
+  <script src="/js/theme-changer.js"></script>
 
   <script>
+	// Enable JS features
```

Comment:
nit: This is tabbed, the rest uses spaces.

Reply:
My bad, I only recently made my editor change tabs to spaces.